enterprise_access/apps/bffs/tests/utils.py

```diff
@@ -34,6 +37,20 @@ def setUp(self):
         }
         self.request.data = {}
 
+        # Mock HandlerContext
+        self.mock_handler_context = mock.MagicMock(
+            _request=self.request,
+            _status_code=status.HTTP_200_OK,
+            _errors=[],
+            _warnings=[],
+            _enterprise_customer_uuid=self.mock_enterprise_customer_uuid,
+            _enterprise_customer_slug=self.mock_enterprise_customer_slug,
+            _lms_user_id=self.request.user.lms_user_id,
+            _enterprise_features={},
+            _data={},
```

Comment:
`data` is not a private attribute, i.e. `HandlerContext` uses `data` vs. `_data`. Rationale: `data` is still exposed for handlers (e.g., `DashboardHandler` to mutate freely, whereas the other private attributes are read-only).